import-export-cli/utils/structs.go

```diff
@@ -29,8 +29,9 @@ type EnvKeysAll struct {
 // For main_config.yaml
 // To be manually edited by the user
 type MainConfig struct {
-	Config       Config                  `yaml:"config"`
-	Environments map[string]EnvEndpoints `yaml:"environments"`
+	Config         Config                  `yaml:"config"`
+	Environments   map[string]EnvEndpoints `yaml:"environments"`
+	MgwAdapterEnvs map[string]MgwEndpoints `yaml:"mgw_adapter_envs"`
```

Comment:
Can we change json:"mgw_adapter_envs" to camel case to make it consistent with the way we are naming json keys elsewhere?

Reply:
Same explanation as https://github.com/wso2/product-apim-tooling/pull/625#discussion_r580809405